easybuild/easyconfigs/p/PyTorch/USER.md

```diff
@@ -215,10 +391,295 @@ and use the dummy partition `container`, e.g.:
+in a single SquashFS file `$CONTAINERROOT/user-software.squashfs` and after reloading the `PyTorch` module that
+is being used, that file will be mounted in the container and provide `/user-software`. This may improve
+performance of Python in the container and is certainly appreciated by your fellow LUMI users.
+To this end, the module provides the `make-squashfs` script. Try
+
+``` bash
+make-squashfs
+ls $CONTAINERROOT
+```
+
+The second command outputs something along the lines of
+
+```
+bin
+easybuild
+lumi-pytorch-rocm-5.6.1-python-3.10-pytorch-v2.2.0-dockerhash-7392c9d4dcf7.sif
+runscripts
+user-software
+user-software.squashfs
+```
+
+so we see that there is now indeed a file `user-software.squashfs` in that subdirectory.
+We do not automatically delete the `user-software` subdirectory, but you can delete it safely using
+
+```
+rm -rf $CONTAINERROOT/user-software
+```
+
+as it can be reconstructed (except for the file dates) from the SquashFS file using the script
+`uname-squashfs` which is also provided by the module.
+
+Reload the module to let the changes take effect and go again in the container:
+
+``` bash
+module load PyTorch/2.2.0-rocm-5.6.1-python-3.10-singularity-20240315
+singularity shell $SIF
+```
+
+Now try
+
+``` bash
+echo "Test" > /user-software/test.txt
+```
+
+and notice that we can no longer write in `/user-software`.
+
+Installing further packages with `pip` would not fail, but they would not be installed where you expect and instead
+would be installed in your home directory. The `pip` command would warn with
+
+```
+Defaulting to user installation because normal site-packages is not writeable
+```
+
+Try, e.g.,
+
+``` bash
+pip install lightning-utilities
+```
+
+and notice that the package (likely) landed in `~/.local/lib//python3.10/site-packages`:
+
+```
+ls ~/.local/lib//python3.10/site-packages
+```
+
+will among other subdirectories contain the subdirectory `pytorch_lightning` and this is 
+not entirely what we want.
+
+Yet it is still possible to install additional packages by first unsquashing the `user-software.squashfs` file
+with 
+
+``` bash
+unmake-squashfs
+```
+
+(assuming that you had removed the `$CONTAINERROOT/user-software` subdirectory before),
+then deleting the SquashFS file:
+
+``` bash
+rm $CONTAINERROOT/user-software.squashfs
+``` 
+
+and reload the module. Make sure though that you first remove the packages that were accidentally installed
+in `~/.local`.
+
+One big warning is needed here though: **If you do a complete re-install of the module with EasyBuild,
+everything in the installation directory is erased, including your own installation. So just to make sure,
+you may want to keep a copy of the `user-software.squashfs` file elsewhere.**
+
+
+### Automation of the procedure
+
+**Try this procedure preferably from a directory that doesn't contain too many files or subdirectories
+as that may slow down EasyBuild considerably.**
```

Comment:
The heading *Automation of the procedure* stands out well with a Firefox browser. However with Chromium, the bold font seems to be exactly the same both for the heading and for the bolded text around it

Reply:
I noticed that too. It is a problem that can only be solved via the style files I think...